ArduPlane/commands_logic.cpp

```diff
@@ -526,17 +526,31 @@ bool Plane::verify_takeoff()
             // course. This keeps wings level until we are ready to
             // rotate, and also allows us to cope with arbitrary
             // compass errors for auto takeoff
-            float takeoff_course = wrap_PI(radians(gps.ground_course_cd()*0.01f)) - steer_state.locked_course_err;
+            float takeoff_course = wrap_PI(radians(gps.ground_course())) - steer_state.locked_course_err;
```

Comment:
Good opportunity to swap to `ahrs.groundspeed_vector()` to fix the dependency on GPS here.

Reply:
yeah, my branch actually has that. I'll update this branch

Reply:
changed